client/config/firebaseWeb.ts

```diff
@@ -0,0 +1,17 @@
+import { initializeApp } from 'firebase/app';
+import { getAuth } from 'firebase/auth';
+
+// Linkta web app's Firebase configuration
+const firebaseConfig = {
+  apiKey: process.env.FIREBASE_API_KEY,
```

Comment:
you should ensure you have an error catcher in case there's no API Key in the .env file

Reply:
Good catch! Thank you Lucie!

Reply:
I'm a little wary about assigning it if it doesn't exist, I think it would be better to ensure that we have a valid API key before assigning it to the firebaseConfig

Reply:
Right, that's probably safer.  I will change the code to check the api key exists in .env file first then proceed with firebase config.